chapters/annotations.tex

```diff
@@ -817,9 +817,17 @@ \subsubsection{Text}\doublelabel{text}
 outline). The contents inherited from \lstinline!FilledShape! is deprecated, but kept for compatibility reasons.
 
 There are a number of common macros that can be used in the text, and
-they should be replaced when displaying the text as follows:
+they should be replaced when displaying the text as follows (in order such that the earliest ones have precedence,
+and using the longest sequence of identifier characters - alphanumeric and \_):
 
 \begin{itemize}
```

Comment:
Numbered list might be preferred. See also https://github.com/modelica/fmi-standard/issues/650.

Reply:
Good idea, but as I understand that even if it helps a bit in this specific case, it is primarily a generic issue for the entire document - and thus I believe that all lists should be handled in that way in one PR.